backend/src/LibCloud/SqlCompiler.fs

```diff
@@ -537,12 +545,22 @@ let rec lambdaToSql
         | EInt8(_, v) ->
           typecheck $"Int8 {v}" TInt8 expectedType
           let name = randomString 10
-          return $"(@{name})", [ name, Sql.int8 v ], TInt8
+          return $"(@{name})", [ name, Sql.int16 (int16 v) ], TInt8
 
         | EUInt8(_, v) ->
           typecheck $"UInt8 {v}" TUInt8 expectedType
           let name = randomString 10
-          return $"(@{name})", [ name, Sql.bytea [| byte v |] ], TUInt8
+          return $"(@{name})", [ name, Sql.int16 (int16 v) ], TUInt8
```

Comment:
and this one?

Reply:
Yes this also was intentional using `bytea` here was wrong.